runtime/reconcilers/report.go

```diff
@@ -367,3 +369,15 @@ func formatExportFormat(f runtimev1.ExportFormat) string {
 		return f.String()
 	}
 }
+
+func fillTimeRange(tr *runtimev1.TimeRange, t time.Time) *runtimev1.TimeRange {
+	if tr == nil {
+		tr = &runtimev1.TimeRange{}
+	}
+
+	// TODO: does it make sense to override start/end here? Or should we keep the user defined start/end?
+	tr.End = timestamppb.New(t)
+	tr.Start = nil
```

Comment:
We should override end, but not touch start. Long term, I think we'll want to pass `reportTime` into the queries, and let the query implementation decide how to let it override the time range. But for now, just overriding the end gets us the behavior we need.

Reply:
Ya makes sense.